### PR TITLE
Shift std/cert SIG this week to Wed 11:05.

### PR DESCRIPTION
**Pull request by garloff**
Both Alex and Kurt are unavail on Thu, thus the shift to Wed. No big date poll -- let's see who is available. We'll have a short meeting if participation is low.

Signed-off-by: Kurt Garloff <kurt@garloff.de>

### CODE REVIEW

**Review by fkr — APPROVED** [review]
LGTM!

File: team_meetings.yml [hunk 1]
```diff
@@ -281,6 +281,11 @@ events:
       interval:
         weeks: 2
       until: 2022-12-31
+      except_on:
+        - 2022-11-17 14:05:00
+      #also_on:
+        # - 2022-11-16 11:05:00
```
Comment (fkr) [inline team_meetings.yml:287]: Imho this should be removed. but no blocker.

Comment (itrich) [reply team_meetings.yml:287]: +1. But that would be a nice addition to https://github.com/scientific-python/yaml2ics

File: onetimes.yml [hunk 2]
```diff
@@ -254,3 +254,22 @@ events:
       Jitsi server on https://conf.scs.koeln:8443/SCS-Tech
       Dial-In: +49-221-292772-611
     location: "https://conf.scs.koeln:8443/SCS-Tech" 
+  - summary: "SIG Standardization/Certification"
+    begin: 2022-11-15 11:05:00
```
Comment (itrich) [inline onetimes.yml:258]: This should be 2022-11-16 (if the meeting will be on Wednesday as stated in the description)

**Review by itrich — CHANGES_REQUESTED** [review]

File: team_meetings.yml (hunk 1 above)
Comment (garloff) [reply team_meetings.yml:287]: I was wondering whether such a feature exists, searched the yaml2ics docs (which consists of a few examples only AFAICS) and then commented this out, leaving it there for human beings. I'll drop it.
If I find the time, I'll open a feature request for yaml2ics.

File: onetimes.yml (hunk 2 above)
Comment (garloff) [reply onetimes.yml:258]: Oops! Thanks for spotting! Will fix!

**Review by itrich — APPROVED** [review]
LGTM!

**Pull request merged**